package/patterns-yast.spec

```diff
@@ -21,7 +21,7 @@
 Name:           patterns-yast
 
 Version:        20190411
-Release:        0
+Release:        1
```

Comment:
Leave this at 0 and OBS will take care. We never touch it on the repository

Reply:
ok, done that becase in my understanding we need to bump the release to get a new package rebuild. Ok reverting.